src/vaults/VaultInternal.ts

```diff
@@ -25,94 +25,96 @@ class VaultInternal {
   public readonly baseDir: string;
   public readonly gitDir: string;
   public readonly vaultId: VaultId;
+  protected readonly remote: boolean;
```

Comment:
From discussions here https://github.com/MatrixAI/js-polykey/issues/253#issuecomment-966704561, I was expecting this to be some kind of `Remote` type. Is there a reason why this is a boolean instead now?

Reply:
I realised it wouldn't really make a huge difference. I looked at it but it would require some reshuffling to the db. On a similar vein their could be some more restructuring  of the vault metadata that is stored in the db and doing this could be a small optimisation for the future. Ill see if I have time for this tomorrow to implement but the gains from it wouldn't be hugely significant

Reply:
Here Ive just made it simple by settting the remote field when constructed which is simply done through the Vault Manager. This is probably the most robust way for now but when expanding the COW and further systems for the vaults this would have to be changed

Reply:
@joshuakarp can this be resolved?

Reply:
Yeah, this can be resolved now. I've added an issue to potentially consider usage of this at a later date: #309